vendor/aftereffects/mac/OpenColorIO_AE_Dialogs_Cocoa.mm

```diff
@@ -223,21 +223,17 @@ bool ColorSpacePopUpMenu(OCIO::ConstConfigRcPtr config, std::string &colorSpace,
         const char *family = colorSpacePtr->getFamily();
         
         
-        NSString *colorSpacePath = nil;
+        NSMutableArray *pathComponents = [NSMutableArray array];
         
-        if(family == NULL || family == std::string(""))
+        if(family != NULL && family != std::string(""))
         {
-            colorSpacePath = [NSString stringWithUTF8String:colorSpaceName];
-        }
-        else
-        {
-            colorSpacePath = [NSString stringWithFormat:@"%s/%s", family, colorSpaceName];
+            assert(config->getFamilySeparator() == '/');
```

Comment:
This appears to be enforcing the default  `/` family separator. Can this be changed to support any separator?